backoffice/backoffice/management/management/commands/delete_author.py

```diff
@@ -0,0 +1,64 @@
+from django.core.management.base import BaseCommand
+
+from backoffice.authors.models import AuthorWorkflow
+import uuid
+from django.conf import settings
+from opensearchpy import OpenSearch
+from opensearchpy.exceptions import NotFoundError
+
+
+class Command(BaseCommand):
+    """
+    A management command that delete a workflow by its UUID.
+    Removes it both from the database and from the OpenSearch.
+    >> python manage.py create_groups
+    """
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "uuid",  # positional argument, so it’s required
+            type=uuid.UUID,
+            help="A valid UUID (e.g., 123e4567-e89b-12d3-a456-426614174000)",
+        )
+
+    def handle(self, *args, **options):
+        connection = settings.OPENSEARCH_DSL["default"]["hosts"]
+        host, port = (
+            connection.split(":")
+            if isinstance(connection, str)
+            else connection[0].split(":")
+        )
+
+        client = OpenSearch(hosts=[{"host": host, "port": port}])
+
+        try:
+            client.delete(
+                index=settings.OPENSEARCH_INDEX_NAMES["backoffice.authors.documents"],
+                id=options["uuid"],
```

Comment:
You are re-using this value many times. Extract on top like 
```
uuid_value = str(options["uuid"])
```

then we just do 

```
        client.delete(index=index_name, id=uuid_value)
```